src/docs/asciidoc/Analytics.asciidoc

```diff
@@ -331,301 +331,11 @@ datasets$testset[,1:4][1,]
-[source,bash]
-----
-xd:>rich-gauge display testgauge
-----
-
-===== Stock Price Example
-
-In this example, we will track stock prices, which is a more practical example. The data is ingested as JSON strings like 
-
-    {"symbol":"VMW","price":72.04}
-
-
-Create an ingest stream
-
-     xd:> stream create --name stocks --definition "http --port=9090 | file"
-
-Next create the tap, using the transform module to extract the stock price from the payload: 
-
-     xd:> stream create --name stockprice --definition "tap:stream:stocks > transform --expression=#jsonPath(payload,'$.price') | rich-gauge"
-
-Now Post some messages to the ingest stream:
-
-    xd:> http post --target http://localhost:9090 --data {"symbol":"VMW","price":72.04}
-    xd:> http post --target http://localhost:9090 --data {"symbol":"VMW","price":72.06}
-    xd:> http post --target http://localhost:9090 --data {"symbol":"VMW","price":72.08}
-
-Note: JSON fields should be separated by a comma without any spaces. Alternatively, enclose the whole argument to `--data` with quotes and escape inner quotes with a backslash.
-
-Check the gauge:
-
-[source,bash]
-----
-xd:>rich-gauge display stockprice
-----
-
-===== Improved Stock Price Example
-
-In this example, we will track stock prices for selected stocks. The data is ingested as JSON strings like 
-
-    {"symbol":"VMW","price":72.04}
-    {"symbol":"EMC","price":24.92}
-
-The previous example would feed these prices to a single gauge. What we really want is to create a separate tap for each ticker symbol in which we are interested:
-
-Create an ingest stream
-
-     xd:> stream create --name stocks --definition "http --port=9090 | file"
-
-Next create the tap, using the transform module to extract the stock price from the payload: 
-
-     xd:> stream create --name vmwprice --definition "tap:stream:stocks > filter --expression=#jsonPath(payload,'$.symbol')==VMW | transform --expression=#jsonPath(payload,'$.price') | rich-gauge" --deploy
-     xd:> stream create --name emcprice --definition "tap:stream:stocks > filter --expression=#jsonPath(payload,'$.symbol')==EMC | transform --expression=#jsonPath(payload,'$.price') | rich-gauge" --deploy
-
-Now Post some messages to the ingest stream:
-
-    xd:> http post --target http://localhost:9090 --data {"symbol":"VMW","price":72.04}
-    xd:> http post --target http://localhost:9090 --data {"symbol":"VMW","price":72.06}
-    xd:> http post --target http://localhost:9090 --data {"symbol":"VMW","price":72.08}
-
-    xd:> http post --target http://localhost:9090 --data {"symbol":"EMC","price":24.92}
-    xd:> http post --target http://localhost:9090 --data {"symbol":"EMC","price":24.90}
-    xd:> http post --target http://localhost:9090 --data {"symbol":"EMC","price":24.96}
-
-Check the gauge:
-
-[source,bash]
-----
-xd:>rich-gauge display emcprice
-xd:>rich-gauge display vmwprice
-----
-
-==== Accessing Analytics Data over the RESTful API
+=== Accessing Analytics Data over the RESTful API
```

Comment:
the REST API only pertains to counters & gauges, so one could argue it should be a child of said section. Maybe move it to the new file as well?